corehq/apps/es/es_query.py

```diff
@@ -405,6 +428,14 @@ def facets(self):
         results = namedtuple('facet_results', [f.name for f in facets])
         return results(**{f.name: f.parse_result(raw) for f in facets})
 
+    @property
+    @memoized
+    def aggregations(self):
+        aggregations = self.query._aggregations
+        raw = self.raw.get('aggregations', {})
+        results = namedtuple('aggregation_results', [a.name for a in aggregations])
+        return results(**{a.name: a.parse_result(raw) for a in aggregations})
```

Comment:
:+1: In looking in to aggregations, did you get a feel for whether or not we could/should drop facets entirely?  (we could even keep the facets API, but use aggregations behind the scenes).  Just thinking that it'd be nice to have only one way to do things if it looks like the switch would be easy.